cms/models.py

```diff
@@ -127,9 +127,14 @@ class HomePage(Page, CommonProperties):
     def get_context(self, request, *args, **kwargs):
         return {
             **super().get_context(request, *args, **kwargs),
+            "alumni": self.alumni,
+            "catalog": self.catalog,
+            "CSOURCE_PAYLOAD": None,
+            "learning_resources": self.learning_resources,
+            "program_description_section": self.program_description_section,
             "site_name": settings.SITE_NAME,
+            "three_column_image_text_section": self.three_column_image_text_section,
```

Comment:
One very minor suggestion: can you put the variables that are not properties of this class above the other variables that are properties?
```suggestion
            "site_name": settings.SITE_NAME,
            "CSOURCE_PAYLOAD": None,
            "alumni": self.alumni,
            "catalog": self.catalog,
            "learning_resources": self.learning_resources,
            "program_description_section": self.program_description_section,
            "three_column_image_text_section": self.three_column_image_text_section,
```

Reply:
@gsidebo, Thanks for the review.
I have reordered the variables along with adding a comment, kindly have a look now.